substratevm/src/com.oracle.svm.reflect/src/com/oracle/svm/reflect/serialize/hosted/SerializationFeature.java

```diff
@@ -76,109 +57,28 @@
     @Override
     public void duringSetup(DuringSetupAccess a) {
         FeatureImpl.DuringSetupAccessImpl access = (FeatureImpl.DuringSetupAccessImpl) a;
+        SerializationSupport support = new SerializationSupport();
+        ImageSingletons.add(SerializationRegistry.class, support);
```

Comment:
`SerializationBuilder` is responsible for building up the data in `SerializationSupport` thus the `SerializationBuilder` singleton should be registers by the `SerializationBuilder` (like it was before)